support-frontend/assets/components/checkoutForm/checkoutForm.jsx

```diff
@@ -35,6 +35,35 @@ FormSection.defaultProps = {
   noBorder: false,
 };
 
+// Hidden version of form section
+
+type FormSectionHiddenPropTypes = {|
+  title: Option<string>,
+  children: Node,
+  headingSize: HeadingSize,
+  hidden?: boolean,
+  id?: Option<string>,
+|};
+
+const FormSectionHidden = ({
+  children, title, headingSize, hidden, id,
+}: FormSectionHiddenPropTypes) => (
+  <div id={id} className={`component-checkout-form-section ${hidden ? 'component-checkout-form-section--hidden' : ''}`}>
+    {!hidden && (
```

Comment:
I guess it's like a hide by default thing, but to avoid a double negative it might be worth calling it something like `show` and `FormSectionShowHide`